java/java.hints/src/org/netbeans/modules/java/hints/errors/CreateFixBase.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.java.hints.errors;
+
+import java.io.IOException;
+import org.netbeans.api.java.source.ModificationResult;
+import org.netbeans.spi.editor.hints.Fix;
+
+/**
+ *
+ * @author Dusan Balek
+ */
+public abstract class CreateFixBase implements Fix {
```

Comment:
This might deserve some versioning, but there is implementation dependency in java.lsp.server anyway:
```
                    <code-name-base>org.netbeans.modules.java.hints</code-name-base>
                    <build-prerequisite/>
                    <compile-dependency/>
                    <run-dependency>
                        <release-version>1</release-version>
                        <implementation-version/>
                    </run-dependency>
```